nimare/utils.py

```diff
@@ -386,3 +387,200 @@ def uk_to_us(text):
+    space (normalized using templates other than those contained
+    in SPM and FSL) using the tal2icbm transform developed and
+    validated by Jack Lancaster at the Research Imaging Center in
+    San Antonio, Texas.
+    http://www3.interscience.wiley.com/cgi-bin/abstract/114104479/ABSTRACT
+    """
+    # Find which dimensions are of size 3
+    shape = np.array(coords.shape)
+    if all(shape == 3):
+        LGR.info("Input is an ambiguous 3x3 matrix.\nAssuming coords are row " "vectors (Nx3).")
+        use_dim = 1
+    elif not any(shape == 3):
+        raise AttributeError("Input must be an Nx3 or 3xN matrix.")
+    else:
+        use_dim = np.where(shape == 3)[0][0]
+
+    # Transpose if necessary
+    if use_dim == 1:
+        coords = coords.transpose()
+
+    # Transformation matrices, different for each software package
+    icbm_other = np.array(
+        [
+            [0.9357, 0.0029, -0.0072, -1.0423],
+            [-0.0065, 0.9396, -0.0726, -1.3940],
+            [0.0103, 0.0752, 0.8967, 3.6475],
+            [0.0000, 0.0000, 0.0000, 1.0000],
+        ]
+    )
+
+    # Invert the transformation matrix
+    icbm_other = np.linalg.inv(icbm_other)
+
+    # Apply the transformation matrix
+    coords = np.concatenate((coords, np.ones((1, coords.shape[1]))))
+    coords = np.dot(icbm_other, coords)
+
+    # Format the output, transpose if necessary
+    out_coords = coords[:3, :]
+    if use_dim == 1:
+        out_coords = out_coords.transpose()
+    return out_coords
+
+
+@due.dcite(
+    references.LANCASTER_TRANSFORM,
+    description="Introduces the Lancaster MNI-to-Talairach transform, "
+    "as well as its inverse, the Talairach-to-MNI "
+    "transform.",
+)
+@due.dcite(
+    references.LANCASTER_TRANSFORM_VALIDATION,
+    description="Validates the Lancaster MNI-to-Talairach and " "Talairach-to-MNI transforms.",
+)
+def mni2tal(coords):
+    """
+    Convert coordinates from MNI space Talairach space.
+
+    Parameters
+    ----------
+    coords : (X, 3) :obj:`numpy.ndarray`
+        Coordinates in MNI space to convert.
+        Each row is a coordinate, with three columns.
+
+    Returns
+    -------
+    coords : (X, 3) :obj:`numpy.ndarray`
+        Coordinates in Talairach space.
+        Each row is a coordinate, with three columns.
+
+    Notes
+    -----
+    Python version of BrainMap's icbm_other2tal.m.
+    This function converts coordinates from MNI space (normalized using
+    templates other than those contained in SPM and FSL) to Talairach space
+    using the icbm2tal transform developed and validated by Jack Lancaster at
+    the Research Imaging Center in San Antonio, Texas.
+    http://www3.interscience.wiley.com/cgi-bin/abstract/114104479/ABSTRACT
+    """
+    # Find which dimensions are of size 3
+    shape = np.array(coords.shape)
+    if all(shape == 3):
+        LGR.info("Input is an ambiguous 3x3 matrix.\nAssuming coords are row " "vectors (Nx3).")
```

Comment:
```suggestion
        LGR.info("Input is an ambiguous 3x3 matrix.\nAssuming coords are row vectors (Nx3).")
```

Good opportunity to fix this mistake introduced by `black`.